src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -9314,5 +9314,640 @@ public unsafe struct CAPROPVARIANT
             var comp = CreateCompilation(csharp, options: TestOptions.UnsafeDebugDll);
             comp.VerifyDiagnostics();
         }
+
+        [Fact]
+        public void TestUnsafeAliasWithCompilationOptionOff1()
+        {
+            var csharp = @"
+using unsafe X = int*;
+
+class C
+{
+}
+";
+            var comp = CreateCompilation(csharp, options: TestOptions.DebugDll);
+            comp.VerifyDiagnostics(
+                // (2,1): hidden CS8019: Unnecessary using directive.
+                // using unsafe X = int*;
+                Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using unsafe X = int*;").WithLocation(2, 1),
+                // (2,7): error CS0227: Unsafe code may only appear if compiling with /unsafe
+                // using unsafe X = int*;
+                Diagnostic(ErrorCode.ERR_IllegalUnsafe, "unsafe").WithLocation(2, 7));
+        }
+
+        [Fact]
+        public void TestUnsafeAliasWithCompilationOptionOff2()
```

Comment:
Is this test subsumed by `TestUnsafeAliasCSharp11_2` (two tests below, line 9378)?

Reply:
no.  one is testing preview, the other is testing C# 11.